anvil/src/eth/backend/mem/mod.rs

```diff
@@ -842,7 +847,13 @@ impl Backend {
             // update block metadata
             storage.best_number = block_number;
             storage.best_hash = block_hash;
-            storage.total_difficulty = storage.total_difficulty.saturating_add(header.difficulty);
+            // Difficulty is removed and not used after Paris (aka TheMerge). Value is replaced with prevrandao.
+            // https://github.com/bluealloy/revm/blob/1839b3fce8eaeebb85025576f2519b80615aca1e/crates/interpreter/src/instructions/host_env.rs#L27
+            if self.is_eip3675() {
```

Comment:
oh yeah, this makes sense.

nice find.
we can simplify this by only using the else branch, we don't need to update the ttd if eip3675

I think it's even debatable if we should increase ttd at all if me mining a block

however this is the first time that this has been raised so I'm ok with increasing it pre merge

Reply:
sounds good! just made the requested change